cupy/math/window.py

```diff
@@ -0,0 +1,37 @@
+import numpy
+
+from cupy.creation.basic import ones
```

Comment:
Do not import objects directly. Instead import modules. See https://docs.openstack.org/hacking/latest/user/hacking.html#imports
```
from cupy.creation import basic
...
basic.ones(...)
```